### PR TITLE
del bigdl engine in orca pytorch estimator

### LINKED ISSUE #7954: Remove jep backend in orca pytorch estmator
As jep's environment is hard to use and deprecated. And ray and spark backend are better choice, we will remove jep backend.
Tasks:
- [ ] 1. remove jep UT and example tests using orca pytorch estimator.
- [ ] 2. remove related code in orca pytorch estimator.

### PR DESCRIPTION
**Pull request by qiuxin2012**
## Description
del bigdl engine in orca pytorch estimator

### 1. Why the change?
del bigdl engine in orca pytorch estimator, as bigdl to hard to deploy.
fixed #7954 

### 2. User API changes

del bigdl engine in orca pytorch estimator， user couldm't use bigdl backend in  orca pytorch estimator

### 3. Summary of the change 

del bigdl engine in orca pytorch estimator

### 4. How to test?
- [ ] N/A
- [ ] Unit test
- [ ] Application test
- [ ] Document test
- [ ] ...

### CODE REVIEW

**Comment by hkvision** [comment]
Also need to delete JEP related nightly jobs.

File: .github/actions/colab/colab-python-py37-pytorch/action.yml [hunk 1]
```diff
@@ -52,7 +52,7 @@ runs:
         pip install -i https://pypi.org/simple --pre bigdl
         pip install -i ${GONDOLIN_PIP_MIRROR} --trusted-host ${GONDOLIN_TRUSTED_HOST} torch==1.7.1 torchvision==0.8.2
         pip install -i ${GONDOLIN_PIP_MIRROR} --trusted-host ${GONDOLIN_TRUSTED_HOST} six cloudpickle
```
Comment (hkvision) [inline .github/actions/colab/colab-python-py37-pytorch/action.yml:54]: can we also delete these two dependencies?

File: python/orca/example/learn/pytorch/brainMRI/brainMRI.py [hunk 2]
```diff
@@ -203,19 +203,7 @@ def denormalize(images):
 # plot some random training images.
 # You should use jupyter notebook to show the images.
 show_batch(train_loader)
-if args.backend == "bigdl":
```
Comment (hkvision) [inline python/orca/example/learn/pytorch/brainMRI/brainMRI.py:206]: also need to modify the readme of this example

File: scala/orca/src/main/scala/com/intel/analytics/bigdl/orca/net/PythonFeatureSet.scala [hunk 3]
```diff
@@ -46,6 +46,7 @@ import org.apache.spark.storage.StorageLevel
 
 import scala.reflect.ClassTag
 
+@deprecated("don't use this, will be remove in the future")
```
Comment (hkvision) [inline scala/orca/src/main/scala/com/intel/analytics/bigdl/orca/net/PythonFeatureSet.scala:49]: remove -> removed

File: scala/orca/src/main/scala/com/intel/analytics/bigdl/orca/net/TorchLoss.scala [hunk 4]
```diff
@@ -24,6 +24,7 @@ import jep.NDArray
 import org.apache.spark.TaskContext
 import com.intel.analytics.bigdl.dllib.utils.Log4Error
 
+@deprecated("don't use this, will be remove in the future")
```
Comment (hkvision) [inline scala/orca/src/main/scala/com/intel/analytics/bigdl/orca/net/TorchLoss.scala:27]: same as above

File: python/orca/src/bigdl/orca/torch/__init__.py [hunk 5]
```diff
@@ -14,6 +14,7 @@
 # limitations under the License.
 #
 
+# TODO(will delete in 2.2.0)
```
Comment (hkvision) [inline python/orca/src/bigdl/orca/torch/__init__.py:17]: is 2.2.0 correct? Next version is 2.3.0

File: .github/actions/colab/colab-python-py37-pytorch/action.yml (hunk 1 above)
Comment (qiuxin2012) [reply .github/actions/colab/colab-python-py37-pytorch/action.yml]: removed

**Review by hkvision — APPROVED** [review]

**Pull request merged**